web/src/components/schema/Object.vue

```diff
@@ -61,8 +87,20 @@ export default defineComponent({
       );
     });
 
+    function renderListOfObjects(value: any[]) {
+      const newVal:any = _.map(value, (v) => v);
+      return newVal;
+    }
+
+    function isObjectArray(value: unknown) {
+      if (Array.isArray(value) && (value[0] instanceof Object)) {
+        return true;
+      } return false;
```

Comment:
This can be simplified to

```suggestion
      return Array.isArray(value) && (value[0] instanceof Object);
```

Reply:
Also, I have a feeling that this function should be a [type predicate](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates).